### ISSUE TITLE
Send environmental impact survey

### ISSUE THREAD

**Issue by edrushka** [MozillaFoundation/plan#663]
Based on the discussion in #633 and in [our meetings] (https://public.etherpad-mozilla.org/p/social_enivro_reponsibility) we've decided to start by surveying MoFo's to find a meaningful and effective way to approach some of these issues.

Draft questions are [here] (https://docs.google.com/document/d/1H41T5FvOOdcnD89k0AgX6LlWUt3aJR0PtBY4uWzsksE/edit#). Feel free to comment in this ticket if you have thoughts.

cc @cassiemc @secretrobotron @metaphorista @adamlofting @stephwright

**Comment by edrushka** [comment]
Whoa, I went a little crazy on the edits!  Can you please review to make sure I didn't take us too far off track? It's more general now, and less Mozlando-specific, which may not be what we want. https://docs.google.com/document/d/1H41T5FvOOdcnD89k0AgX6LlWUt3aJR0PtBY4uWzsksE/edit#

**Comment by stephwright** [comment]
I love your edits!  They clarified questions and kept them to obtaining information with which we can actually do something!  Nicely done!!! :clap:

**Comment by OpenMatt** [comment]
Would be nice to have a link or something to share on this in Jan 25 staff meeting

**Comment by cassiemc** [comment]
@OpenMatt shouldn't be a problem, our aim was to send this out by end of next week, and we can use the 25th meeting as a reminder. Thanks!

**Comment by edrushka** [comment]
I had a convo with @davidascher about this survey and he shared some really great insights and suggestions. Would love to discuss those with the group before we send this out.

**Comment by cassiemc** [comment]
@edrushka and I chatted and decided to try and rewrite the survey a bit. I'll share it at @davidascher's staff call next week for some feedback, and we can decide later as a team if/when to send it out. Postponing to next heartbeat.

**Comment by cassiemc** [comment]
@edrushka Can you help us set up another green team meeting? I think we need that before we can close this. Going to modify the milestone for next heartbeat.

**Comment by edrushka** [comment]
Yes! Let's circle up and chat about this survey. I have some updates from Hive Chicago Buzz to report as well.

**Comment by cassiemc** [comment]
We decided not to do this in the end. We intend to move on the plenty of ways we can have an immediate impact, particularly with @edrushka's new events role. Some other green team ideas/initiatives will be filed here: https://github.com/MozillaFoundation/green/issues
